### DISCUSSION TITLE
How to change Watermark style

### DISCUSSION THREAD

**Discussion by kzanderoff** [MahApps/MahApps.Metro#3996] [Q&A] [answered]
How to change watermark foreground/opacity or override style ?

**Comment by punker76** [comment]
@kzanderoff The foreground can be changed with the `MahApps.Brushes.TextBlock.FloatingMessage` brush and the font size with the `MahApps.Font.Size.FloatingWatermark` resource key.

**Reply by punker76** [reply]
Example with the current `develop` resource keys:

```xaml
<Application.Resources>
    <SolidColorBrush x:Key="MahApps.Brushes.TextBlock.FloatingMessage" Color="OrangeRed" />
    <x:Double x:Key="MahApps.Font.Size.FloatingWatermark">11</x:Double>
</Application.Resources>

<TextBox mah:TextBoxHelper.UseFloatingWatermark="True"
         mah:TextBoxHelper.Watermark="User name" />
```

If you also want to change opacity, override `MahApps.Styles.TextBlock.Watermark` and/or `MahApps.Styles.TextBlock.AutoCollapsing`, because both are defined with `Opacity="0.6"` on current `develop`.
